exposed-tests/src/main/kotlin/org/jetbrains/exposed/sql/tests/DatabaseTestsBase.kt

```diff
@@ -21,12 +21,7 @@ enum class TestDB(
     var db: Database? = null
 ) {
     H2({ "jdbc:h2:mem:regular;DB_CLOSE_DELAY=-1;" }, "org.h2.Driver"),
-    H2_MYSQL(
-        { "jdbc:h2:mem:mysql;MODE=MySQL;DB_CLOSE_DELAY=-1" }, "org.h2.Driver",
-        beforeConnection = {
-            Mode.getInstance("MySQL").convertInsertNullToZero = false
```

Comment:
This option is removed in `200`, see: https://github.com/h2database/h2database/issues/1942